### ISSUE TITLE
Version 1.8 ambiguity

### ISSUE THREAD

**Issue by AMDmi3** [lvc/pkgdiff#35]
Version 1.8 is mentioned in the changelog and in the README, however there's no tag and it's not mentioned in the _Downloads_. Please either add a tag and download for 1.8 or do not mention non-existing version, as it creates confusion for users: https://repology.org/metapackage/pkgdiff/versions

**Comment by sergiomb2** [comment]
yes, I notice this today, you announced version 1.8 but we don't have any tag
